control_panel_api/models.py

```diff
@@ -234,24 +234,37 @@ class Meta:
     def aws_role_name(self):
         return self.app.iam_role_name
 
+    def has_readwrite_access(self):
+        return self.access_level == self.READWRITE
+
 
 class UserS3Bucket(AccessToS3Bucket):
     """
     A user can have access to several S3 buckets.
 
     We have two access levels, "readonly" (default) and "readwrite".
-    The `is_admin` field determine if the user has admin privileges on the
-    S3 bucket
-    """
 
+    The `is_admin` field determines the `access_level` to the S3Bucket. The
+    `access_level` can not be changed directly.
+    """
     user = models.ForeignKey(
         User, related_name='users3buckets', on_delete=models.CASCADE)
     is_admin = models.BooleanField(default=False)
```

Comment:
I feel field name doesn't quite fit - it should be `user_is_admin`

Reply:
Well it's on the USERS3Bucket so this isn't an issue to me.  @xoen ?